preview/example.js

```diff
@@ -1,5 +1,10 @@
+var fs = require('fs');
+var insertStyle = require('insert-css');
 
-b = require('boxbits');
+var css = fs.readFileSync(__dirname + '/reset.css', 'utf8');
+insertStyle(css);
+
+var b = require('../');
```

Comment:
The example file should require `boxbits` as an external lib, not a relative reference, just like a typical project would